simapp/app_legacy.go

```diff
@@ -299,6 +299,9 @@ func NewSimApp(
 	app.GroupKeeper = groupkeeper.NewKeeper(keys[group.StoreKey], appCodec, app.MsgServiceRouter(), app.AccountKeeper, groupConfig)
 
 	// register the proposal types
+	// Note: avoid adding new handlers, instead use the new proposal flow
+	// by granting the governance module the right to execute the message.
+	// See: https://github.com/cosmos/cosmos-sdk/blob/release/v0.46.x/x/gov/spec/01_concepts.md#proposal-messages
```

Comment:
I was even thinking to change Note -> Deprecated.

Reply:
Note sgtm.

Reply:
+1 for deprecated, old gov handlers are deprecated